lib/browser/guest-view-manager.ts

```diff
@@ -245,23 +260,25 @@ const attachGuest = function (event, embedderFrameId, elementInstanceId, guestIn
   guestInstance.elementInstanceId = elementInstanceId;
 
   watchEmbedder(embedder);
-
-  webViewManager.addGuest(guestInstanceId, elementInstanceId, embedder, guest, webPreferences);
+  if (webViewManager) {
+    webViewManager.addGuest(guestInstanceId, elementInstanceId, embedder, guest, webPreferences);
+  }
   guest.attachToIframe(embedder, embedderFrameId);
 };
 
 // Remove an guest-embedder relationship.
-const detachGuest = function (embedder, guestInstanceId) {
+const detachGuest = function (embedder: WebContents, guestInstanceId: number) {
   const guestInstance = guestInstances[guestInstanceId];
 
   if (!guestInstance) return;
 
   if (embedder !== guestInstance.embedder) {
     return;
   }
-
-  webViewManager.removeGuest(embedder, guestInstanceId);
-  delete guestInstances[guestInstanceId];
+  if (webViewManager) {
```

Comment:
We should assert that `webViewManager` exists here instead of truthy checking, we want to do `webViewManager!` and let it error if it doesn't exist, it's a theoretical impossible case.